core/src/main/java/org/testcontainers/utility/MountableFile.java

```diff
@@ -361,6 +366,31 @@ public String getDescription() {
         return this.getResolvedPath();
     }
 
+    @Override
+    public void updateChecksum(Checksum checksum) {
+        byte[] pathBytes = getBytes();
+        // Add path to the hash
+        checksum.update(pathBytes, 0, pathBytes.length);
+
+        File file = new File(getResolvedPath());
```

Comment:
This is wrongly copied from `GenericContainer`, there are no "path bytes" here:

```suggestion
        File file = new File(getResolvedPath());
```

Reply:
lol true funny that it doesn't change the test outcome X-D

Reply:
done